eventbrite.py

```diff
@@ -0,0 +1,380 @@
+import urllib3
+import json
+import os
+
+# EVENTBRITE API SCRIPT
+
+# Retrieve the values of an environment variables
+eventbrite_api_token = os.environ.get('EVENTBRITE_API_TOKEN')
+organization_id = os.environ.get('EVENTBRITE_ORGANISATION_ID')
+event_id = os.environ.get('EVENTBRITE_EVENT_ID')
+ticket_class_id = os.environ.get('EVENTBRITE_TICKET_CLASS_ID')
```

Comment:
Remove event_id, and ticket_class_id, these will be returned by your create event/ticket functions.

Reply:
In fact I want all these `os.environ` calls to be run at the bottom where `if __name == '__main__'` is. Pass those values as arguments to your creation functions.
eg.
```py
def create_function(http, api_token, organisation_id, ...)
```